js/src/utils/createMessageForMultipleErrors.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf, _n, _x } from '@wordpress/i18n';
+
+/**
+ * Creates a message for a list of error messages. For example for a list of error messages: createMessageForMultipleErrors(['Target Audience', 'Shipping Rates']) will create the following message:
+ * 'There are errors in the following actions: Target Audience and Shipping Rates. Please try again later.'
+ *
+ * @param {Array<string>} errorMessages Error messages to be joined.
+ * @param {boolean} [isPartiallySuccessful='true'] Whether some actions were successful.
+ * @return {string|null} - Return the error message for a list of errors otherwise null if there are no error messages
+ */
+export default function createMessageForMultipleErrors(
+	errorMessages,
+	isPartiallySuccessful = true
+) {
+	if ( errorMessages.length ) {
+		const separator = _x(
+			', ',
+			'the separator for concatenating the messages of failed actions',
+			'google-listings-and-ads'
+		);
+		const listErrors = sprintf(
+			// translators: 1: optional string when there are multiple failed actions, and it's a concatenated text of failed actions except for the last one. 2: the last one or the only failed action.
+			_n(
+				'There is an error in the following action: %1$s%2$s.',
```

Comment:
:nail_care: 
`%1S` is always `''` isn't it? Personally, I'd avoid one interpolation, 
```suggestion
				'There is an error in the following action: %2$s.',
```

Reply:
Yes, it's always `''` but the eslint is unhappy about it. 🤷‍♂️
![image](https://user-images.githubusercontent.com/17420811/184307553-ce189c73-ef73-4bbf-950e-9bc008addccd.png)

Reply:
I don't quite understand why we have such a rule, but let's keep it then. :)